### PR TITLE
feat: Change ROS version to Noetic

### PR DESCRIPTION
**Pull request by alanssitis**
## What is a quick description of the change?
Update ROS version to Noetic

## Is this fixing an issue?
Not an open one

## Were any issues created as a result of this change?
None which is known

## Are there more details that are relevant?
Due to the fact that there doesn't exist an official "desktop" Noetic image, had to specify stage_ros as a dependency. 

## Check lists (check x in [ ] of list items)
For each of these items, please refer to the [software style guide](https://github.com/Autonomous-Motorsports-Purdue/AMP_ASSv2/wiki/Software-Style-Guide)
- [ ] Test written/updated
- [ ] Tests passing
- [ ] Logging (where appropriate)
- [ ] Coding style

### CODE REVIEW

File: src/env_sim/launch/stage.launch [hunk 1]
```diff
@@ -18,15 +17,6 @@
 	<node pkg="slam_mode_goal" type="slam_mode_goal.py" name="slam_mode_goal" output="screen">
 	</node>
 
-
-	<!-- VISUALIZATION -->
-	<node name="rviz" pkg="rviz" type="rviz" />
```
Comment (zghera) [inline src/env_sim/launch/stage.launch:23]: Why did you remove this?

**Review by zghera — CHANGES_REQUESTED** [review]

File: src/env_sim/launch/stage.launch (hunk 1 above)
Comment (alanssitis) [reply src/env_sim/launch/stage.launch:23]: Rviz was not being used. It was only launched and it did not display anything.

Comment (zghera) [reply src/env_sim/launch/stage.launch:23]: So with RViz in this launch file, there was no window that popped up with a visualization of the kart going around the 2D track? If so, that is really weird and I'll take a look. But if it did show up, then I think we should keep this as we want to see the visualization.

Comment (alanssitis) [reply src/env_sim/launch/stage.launch:23]: A window does pop up, but it does not display anything. Here's a screenshot of it. It is not being used by anything and only the window to the left is useful, and it is controlled by stage_ros.

Screenshot from 2022-02-15 18-09-53

**Review by zghera — APPROVED** [review]
